packages/react-dom-bindings/src/client/ReactDOMComponentTree.js

```diff
@@ -288,10 +288,16 @@ export function getResourcesFromRoot(root: HoistableRoot): RootResources {
   return resources;
 }
 
-export function isMarkedResource(node: Node): boolean {
-  return !!(node: any)[internalResourceMarker];
+export function isMarkedHoistable(node: Node): boolean {
```

Comment:
renamed to reference the more general term for hoistables. should have done that in the last PR